packages/react-components/src/templates/__tests__/WorkingGroupHeader.test.tsx

```diff
@@ -0,0 +1,49 @@
+import { render } from '@testing-library/react';
+
+import WorkingGroupHeader from '../WorkingGroupHeader';
+
+const baseProps = {
```

Comment:
Base props should fill out the absolute minimum for the component to render. Ideally all text strings should be '' and then overwritten and asserted on in a test to be as clear as possible in what your putting into the component and what your expecting out. In an ideal world changing base props would never break a test and in turn make the tests less brittle